src/harness/testcases/registration_testcase.py

```diff
@@ -792,12 +796,15 @@ def test_WINNF_FT_S_REG_18(self):
     devices = [device_a, device_b, device_c]
     for device in devices:
         self._sas_admin.InjectFccId({'fccId': device['fccId']})
+        device['measCapability'] = []
     request = {'registrationRequest': devices}
     response = self._sas.Registration(request)['registrationResponse']
     # Check registration response
+    cbsd_ids = []
```

Comment:
I don't think you actually use this variable (except to write to it). Remove this line and line 807?